graphql/resolve/mutation_rewriter.go

```diff
@@ -1383,11 +1383,34 @@ func rewriteObject(
 				// TODO: Add a function for parsing idVal. This is repeatitive
```

Comment:
You may remove this TODO after addressing Abhimanyu's comment on writing extractVal function.